pkg/image/content_helpers.go

```diff
@@ -8,43 +8,20 @@ import (
 	"github.com/anchore/stereoscope/pkg/filetree"
 )
 
-// fetchFileContentsByPath is a common helper function for resolving the file contents for a path from the file
+// fetchReaderByPath is a common helper function for resolving the file contents for a path from the file
 // catalog relative to the given tree.
-func fetchFileContentsByPath(ft *filetree.FileTree, fileCatalog *FileCatalog, path file.Path) (io.ReadCloser, error) {
-	exists, fileReference, err := ft.File(path, filetree.FollowBasenameLinks)
+func fetchReaderByPath(ft filetree.Reader, fileCatalog FileCatalogReader, path file.Path) (io.ReadCloser, error) {
```

Comment:
I'm 👍 on this naming change here, but it might be easier to review with fewer "cleanup" changes